home/templates/pages/Contactus.html

```diff
@@ -30,78 +35,89 @@ <h1>Contact us</h1>
-                required
-              />
+
+  <!-- Contact Form and Map Section -->
+  <section class="contact-section">
+    <div class="container">
+      <div class="row align-items-center">
+        <!-- Contact Form -->
+        <div class="col-12 col-md-6 contact-form-container" id="contact-form-container">
+          <h2 class="text-center">Leave Us a Message</h2>
+          <form
+            id="contact-form"
+            action="/contact-central"
+            method="post"
+            onsubmit="handleSubmit(event)"
+          >
+            <div class="form-group">
+              <label for="name">Full Name</label>
+              <input type="text" class="form-control" id="name" name="name" placeholder="Enter your name" required />
             </div>
-          </div>
-          <div class="form-group">
-            <label for="email">Email</label>
-            <input
-              type="email"
-              class="form-control"
-              id="email"
-              name="email"
-              placeholder="Enter your email"
-              required
-            />
-          </div>
-          <div class="form-group">
-            <label for="message">Message</label>
-            <textarea
-              class="form-control"
-              id="message"
-              name="message"
-              rows="4"
-              placeholder="Enter your message"
-            ></textarea>
-          </div>
-          <div class="pt-3 text-center">
-            <button type="submit" class="btn btn-primary">Submit</button>
-          </div>
-          {% csrf_token %}
-        </form>
+            <div class="form-group">
+              <label for="email">Email Address</label>
+              <input type="email" class="form-control" id="email" name="email" placeholder="Enter your email" required />
+            </div>
+            <div class="form-group">
+              <label for="message">Message</label>
+              <textarea class="form-control" id="message" name="message" rows="4" placeholder="Enter your message"></textarea>
+            </div>
+            <div class="text-center">
+              <button type="submit" class="btn btn-primary">Send Message</button>
+            </div>
+            {% csrf_token %}
+          </form>
+        </div>
+
+        <!-- Google Map with Address -->
+        <div class="col-12 col-md-6 map-container" id="map-container">
+          <iframe
+            src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3151.8955021029794!2d145.1213769157944!3d-37.84991947974343!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x6ad6419eb35a09f1%3A0x7e3ed94e3ebf12d!2s221%20Burwood%20Hwy%2C%20Burwood%20VIC%203125!5e0!3m2!1sen!2sus!4v1697838690523"
+            width="100%"
+            height="450"
+            style="border:0;"
+            allowfullscreen=""
+            loading="lazy"
+            referrerpolicy="no-referrer-when-downgrade"
+          ></iframe>
```

Comment:
missing CSP for iframe prevents it from loading
![image](https://github.com/user-attachments/assets/3d4b5dd4-709a-4ca8-9e49-590013fe6926)